api/api.go

```diff
@@ -119,6 +119,7 @@ func (a *API) Register(r *mux.Router) {
 	resr.HandleFunc("/users", a.addUser).Methods("OPTIONS", "POST")
 	resr.HandleFunc("/users/{user:[a-z 0-9]+}", a.removeUser).Methods("OPTIONS", "DELETE")
 	resr.HandleFunc("/users", a.listUsers).Methods("OPTIONS", "GET")
+	resr.HandleFunc("/users/{user:[a-z 0-9]+}/change_password", a.changePassword).Methods("OPTIONS", "PUT")
```

Comment:
```suggestion
	resr.HandleFunc("/users/{user:[a-z 0-9]+}", a.changePassword).Methods("OPTIONS", "PUT")
```

this is more reasonable

Reply:
> this is more reasonable

IMO, we should specify the **action**, in this case - `change password`.